app/models/course.rb

```diff
@@ -323,15 +314,69 @@ class Course < ActiveHash::Base
             'You are an intermediate-level developer',
             'You might be familiar with version control and basic web architecture'
           ]
+        }
+      ]
+    }, {
+      id: 8,
+      name: 'BWAG Go Portugues',
+      title: 'Building Web Apps With Go - Em Portugues',
+      description: "Se você está lendo isto então você está prestes a embarcar em uma jornada de iniciante a pro. Sério! Programação web em Go é tão divertido e fácil que você nem vai perceber quanta informação você está aprendendo ao longo do curso. Todo o material do curso está aqui: <a href='https://gobridge.gitbooks.io/building-web-apps-with-go/content/pt-br/index.html'>https://gobridge.gitbooks.io/building-web-apps-with-go/content/pt-br/index.html</a>.",      levels: [
+        {
+          level: 2,
+          color: 'green',
+          title: "Mais ou Menos Novato(a) em Programação",
+          level_description: [
+            'Você sabe o que é uma função',
+            'Talvez você já tenha feito um tutorial ou dois online',
+            'Você sabe usar o básico do terminal - mudar de diretórios, por exemplo'
+          ]
         }, {
-          level: 5,
-          color: 'purple',
-          title: "Ready for the Next Challenge",
+          level: 3,
+          color: 'gold',
+          title: "Alguma Experiência em Go",
           level_description: [
-            'You\'ve gone through the <a href="http://tour.golang.org/welcome/1">Go Tour</a>.',
-            'You\'re comfortable with the terminal',
-            'You want to problem-solve instead of copying other\'s code',
-            'You want to build an app with minimum guidance'
+            'Você sabe como definir uma função em Go',
+            'Você entende bem como funciona slices e maps em Go',
+            'Você tem um entendimento básico sobre a estrutura de um aplicativo Go, talvez de outro workshop ou tutorial',
+            'Você se sente a vontade no terminal, mas não é necessariamente um Super Usuário(a)'
+          ]
+        }, {
+          level: 4,
+          color: 'orange',
+          title: "Outras Experiências em Programação",
+          level_description: [
+            'Você não sabe Go',
+            'Você é proficiente em outra linguagem e entende conceitos gerais de programação, como por exemplo coleções e escopo',
+            'Você é desenvolvedor(a) de nível intermediário',
+            'Talvez você saiba o básico sobre controle de versão e arquitetura web básica'
+          ]
+        }
+      ]
+    }, {
+      id: 9,
+      name: 'Hardcore Go',
```

Comment:
I know I should be giving a code review and not a review of the class names. But I want to flag that "Hardcore Go" as a class title sounds pretty alienating to me. I wouldn't go because I would expect a mashup of chest bumps and Chuck Norris jokes. However, the description of the class, "This class provides an intensive, comprehensive and idiomatic view of Go," sounds really interesting. What are your thoughts on alternatives like "Idiomatic Go" or "The Go Master Class"?